website/content/api-docs/auth/kubernetes.mdx

```diff
@@ -37,11 +37,18 @@ access the Kubernetes API.
   JWTs. If a certificate is given, its public key will be
   extracted. Not every installation of Kubernetes exposes these
   keys.
-- `issuer` `(string: "")` - Optional JWT issuer. If no issuer is specified, then this plugin will
   use `kubernetes/serviceaccount` as the default issuer. See [these instructions](/docs/platform/k8s/csi#setting-issuer-for-kubernetes-authentication) for looking up the issuer for a given Kubernetes cluster.
-- `disable_iss_validation` `(bool: false)` - Disable JWT issuer validation. Allows to skip ISS validation.
 - `disable_local_ca_jwt` `(bool: false)` - Disable defaulting to the local CA cert and service account JWT when running in a Kubernetes pod.
 
+### Deprecated Parameters
+
+The following fields have been deprecated and will be removed in a future release:
+
+* `disable_iss_validation` `(bool: true)` Disable JWT issuer validation. Allows to skip ISS validation.
+
+
+* `issuer` `(string: "")` Optional JWT issuer. If no issuer is specified, then this plugin will
```

Comment:
It looks like the rest of the `issuer` sentences need to be moved here as well.

Reply:
I feel like it should also mention that `issuer` has been deprecated and will be removed in a future release.